source/includes/examples/ex-display-line-numbers.rst

```diff
@@ -1,5 +1,5 @@
-This code will dynamically update the :binary:`~bin.mongosh` prompt
-to display line numbers:
+To update the :binary:`~bin.mongosh` prompt to display line numbers, run
```

Comment:
[suggestion]

Consider, "To display line numbers in the :binary:`~bin.mongosh` prompt, run..."